### DISCUSSION TITLE
Couple questions on phases and multi pole breakers

### DISCUSSION THREAD

**Discussion by eblohm** [emporia-vue-local/esphome#322] [Q&A]
Bought 2 Vue 3s in their black friday combo pack and had a couple questions on setting up the yaml file - 

1. I've got a 200amp single phase panel, but have 3 wires coming in (2 energized, 1 neutral) are both of the energized wires considered their own phase for the Vue 3? I believe that's the case, but I'd rather not have to open the Vue 3 back up once I've got everything flashed and set up.
2. I bought 2 so I could monitor every breaker in my 30 breaker box. For my multi-pole breakers, do I set those up as if each one is it's own circuit? (Ie, I have 1 50amp breaker in spots 19/21 on my panel, do I list circuit 19 and 21 separately and exclude the multiplier?) 

Thanks!

Side note - anyone have any tips for opening up the smart outlets so I can flash tasmota?

**Comment by flaviut** [comment]
> I'd rather not have to open the Vue 3 back up

absolutely not a problem. You can reflash it wirelessly all you want once you do the initial flash. There's no stress at all about getting it right the first time.

> For my multi-pole breakers, do I set those up as if each one is it's own circuit?

You've got a good plan here! You can also try and get fancy with template sensors to add things together.

> anyone have any tips for opening up the smart outlets

My recollection is that you're better off with pretty much any other smart outlet when it comes to reflashing.
